### PR TITLE
Update GetGasPrice RPC endpoint with global `MinGasPrice`

### PR DESCRIPTION
**Pull request by crypto-facs**
## Description

This PR aims to update GetGasPrice API with the new GlobalMinGasPrice module param. The idea is to return this value as minimum for transactions to adjust to the new decorator.

______

For contributor use:

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

**Review by loredanacirstea — APPROVED** [review]
utACK

File: rpc/backend/evm_backend.go [hunk 1]
```diff
@@ -818,6 +818,14 @@ func (b *Backend) BaseFee(height int64) (*big.Int, error) {
 	return res.BaseFee.BigInt(), nil
 }
 
+func (b *Backend) GlobalMinGasPrice() (sdk.Dec, error) {
```
Comment (fedekunze) [inline rpc/backend/evm_backend.go:821]: can you add a comment to this function?

File: rpc/namespaces/ethereum/eth/api.go [hunk 2]
```diff
@@ -211,6 +211,16 @@ func (e *PublicAPI) GasPrice() (*hexutil.Big, error) {
...
 
+	// return at least GlobalMinGasPrice from FeeMarket module
+	minGasPrice, err := e.backend.GlobalMinGasPrice()
+	if err != nil {
+		return nil, err
+	}
+	minGasPriceInt := minGasPrice.BigInt()
+	if result.Cmp(minGasPriceInt) < 0 {
+		result = minGasPriceInt
+	}
```
Comment (fedekunze) [inline rpc/namespaces/ethereum/eth/api.go:222]: this looks good! can you update the function comment to describe the new logic?

**Review by fedekunze — COMMENTED** [review]
LGTM! Can you update the changelog too? 🙏

**Review by fedekunze — APPROVED** [review]

**Review by alexanderbez — APPROVED** [review]

**Pull request merged**